pom.xml

```diff
@@ -92,6 +93,11 @@
 			</dependency>
 
 			<!-- Solving convergence errors -->
+			<dependency>
```

Comment:
Since spring boot 3.0 pretty much migrates to the jakarta libs and the jakarta libs will be included anyway I think it is better to migrate in our code as well instead of hanging on to javax.

In fact in my own private projects 90% of my migration to spring boot 3.0 was replacing "import javax." with "import jakarta."

Reply:
I don't disagree - but problem here is that the error is **triggered by** spring-boot-starter-data-jpa:

```
Dependency convergence error for javax.annotation:javax.annotation-api:jar:1.3:runtime paths to dependency are:
+-org.dcsa.shared-kernel:dcsa-shared-kernel-domain:jar:0.3.0-spring-boot-3-SNAPSHOT
  +-org.springframework.boot:spring-boot-starter-data-jpa:jar:3.0.0:compile
    +-org.hibernate.orm:hibernate-core:jar:6.1.5.Final:compile
      +-org.jboss.logging:jboss-logging:jar:3.5.0.Final:runtime
        +-org.jboss.logmanager:jboss-logmanager:jar:2.1.18.Final:runtime
          +-org.wildfly.common:wildfly-common:jar:1.5.1.Final:runtime
            +-javax.annotation:javax.annotation-api:jar:1.3:runtime
and
+-org.dcsa.shared-kernel:dcsa-shared-kernel-domain:jar:0.3.0-spring-boot-3-SNAPSHOT
  +-org.springframework.boot:spring-boot-starter-data-jpa:jar:3.0.0:compile
    +-org.springframework.data:spring-data-jpa:jar:3.0.0:compile
      +-com.querydsl:querydsl-apt:jar:jakarta:5.0.0:provided
        +-com.querydsl:querydsl-codegen:jar:5.0.0:provided
          +-com.querydsl:querydsl-core:jar:5.0.0:provided
            +-javax.annotation:javax.annotation-api:jar:1.3.2:provided
and
+-org.dcsa.shared-kernel:dcsa-shared-kernel-domain:jar:0.3.0-spring-boot-3-SNAPSHOT
  +-org.springframework.boot:spring-boot-starter-data-jpa:jar:3.0.0:compile
    +-org.springframework.data:spring-data-jpa:jar:3.0.0:compile
      +-com.querydsl:querydsl-apt:jar:jakarta:5.0.0:provided
        +-com.querydsl:querydsl-codegen:jar:5.0.0:provided
          +-javax.annotation:javax.annotation-api:jar:1.3.2:runtime
```

Reply:
I see :)